README.md

```diff
@@ -18,20 +18,12 @@ Karavi Metrics for PowerFlex is part of the Karavi open source suite of Kubernet
 [![Go version](https://img.shields.io/github/go-mod/go-version/dell/karavi-metrics-powerflex)](go.mod)
 [![Latest Release](https://img.shields.io/github/v/release/dell/karavi-metrics-powerflex?label=latest&style=flat-square)](https://github.com/dell/karavi-metrics-powerflex/releases)
 
-Karavi Metrics for PowerFlex is an open source distributed solution that provides standardized approaches to gaining observability into Dell EMC products. This project provides the following metrics:
+Karavi Metrics for PowerFlex is an open source distributed solution that provides standardized approaches to gaining observability into Dell EMC PowerFlex. This project provides the following metrics:
```

Comment:
I think "approaches to gain observability", instead of "gaining", is grammatically correct.